locale/en/general.cfg

```diff
@@ -17,6 +17,7 @@ tycoon-gui-basic-needs-met=Basic needs met
 tycoon-gui-basic-needs=Basic Needs
 tycoon-gui-citizens=Citizens
 tycoon-gui-construction-sites=Construction Sites
+tycoon-gui-debt=Unpaid debt
```

Comment:
```suggestion
tycoon-gui-debt=Pending payouts
tycoon-gui-debt-tooltip=When treasuries are full, the city keeps track of pending payouts. Make space in the city's treasury to release the payout.
```

Reply:
What do you think of "pending payouts" instead of "unpaid debt"?

Reply:
this looks really good!